drizzlepac/sky.py

```diff
@@ -39,53 +40,438 @@
+          (see NOTES below). For a typical use, it will compute
+          sky values for each chip/image extension (marked for sky
+          subtraction in the :py:obj:`input` parameter) in an input image,
+          and it will subtract the previously found minimum sky value
+          from all chips (marked for sky subtraction) in that image.
+          This process is repeated for each input image.
+
+          .. note::
+              This setting is recommended when regions of overlap between images
+              are dominated by "pure" sky (as opposite to extended, diffuse
+              sources). This is similar to the "skysub" algorithm used in previous
+              versions of astrodrizzle.
+
+        * *globalmin* : compute a common sky value for all members of
+          *all exposures* (see NOTES below). It will compute
+          sky values for each chip/image extension (marked for sky
+          subtraction in the ``input`` parameter) in all input
+          images, find the minimum sky value, and then it will
+          subtract the same minimum sky value from all chips
+          (marked for sky subtraction) in all images. This method *may*
+          useful when input images already have matched background values.
+
+        * *match* : compute differences in sky values between images
+          in common (pair-wise) sky regions. In this case computed sky values
+          will be relative (delta) to the sky computed in one of the
+          input images whose sky value will be set to (reported to be) 0.
+          This setting will "equalize" sky values between the images in
+          large mosaics. However, this method is not recommended when used
+          in conjunction with `AstroDrizzle <http://stsdas.stsci.edu/stsci_python_sphinxdocs_2.13/drizzlepac/astrodrizzle.html>`_
+          because it computes relative sky values while ``AstroDrizzle`` needs
+          "measured" sky values for median image generation and CR rejection.
+
+        * *globalmin+match* : first find a minimum "global" sky value
+          in all input images and then use 'match' method to
+          equalize sky values between images.
+
+          .. note::
+              This is the *recommended* setting for images
+              containing diffuse sources (e.g., galaxies, nebulae)
+              covering significant parts of the image.
+
+    skywidth : float, optional (Default Value = 0.1)
+        Bin width, in sigma, used to sample the distribution of pixel flux values 
+        in order to compute the sky background statistics.
+
+
+    skystat : {'median', 'mode', 'mean'}, optional (Default Value = 'median')
+        Statistical method for determining the sky value from the image pixel values.
+
+
+    skylower : float, optional (Default Value = INDEF)
+        Lower limit of usable pixel values for computing the sky. This value 
+        should be specified in the units of the input image.
+
+
```

Comment:
Nitpick: remove at least one empty line.

Reply:
fixed in [bb304c9dc2249be45a44a554aa5b9ad0ebfa77ec](https://github.com/spacetelescope/drizzlepac/pull/2079/commits/bb304c9dc2249be45a44a554aa5b9ad0ebfa77ec)